sources/api/certdog/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "certdog"
+version = "0.1.0"
+authors = ["Arnaldo Garcia Rincon <agarrcia@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2018"
+publish = false
+build = "build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[dependencies]
+apiclient = { path = "../apiclient" }
+argh = "0.1.3"
+base64 = "0.13"
+http = "0.2"
+log = "0.4"
+models = { path = "../../models" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1"
+simplelog = "0.10"
+snafu = "0.6"
+tokio = { version = "1", default-features = false, features = ["macros", "rt-multi-thread"] }
+x509-parser = "0.9.2"
+tempfile = "3.2.0"
+
+[build-dependencies]
+cargo-readme = "3.1"
```

Test-only dependencies can go in `dev-dependencies` and they are not compiled during non-test builds.

```suggestion
x509-parser = "0.9.2"

[build-dependencies]
cargo-readme = "3.1"

[dev-dependencies]
tempfile = "3.2.0"
```